manifest.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0" xmlns:ov="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="TaskPaneApp">
+  <Id>99ea9ad2-971a-46ad-99d1-190379478cb9</Id>
+  <Version>1.0.0.0</Version>
+  <ProviderName>Accord Project</ProviderName>
+  <DefaultLocale>en-US</DefaultLocale>
+  <DisplayName DefaultValue="Accord Project"/>
+  <Description DefaultValue="Contract editor for the Accord Project Ecosystem"/>
+  <IconUrl DefaultValue="https://localhost:3000/assets/ap-48.png"/>
```

Comment:
Do we want `localhost` hardcoded in here?

Reply:
I am not sure about this. I will try omitting it and see if it works.

Reply:
@irmerk simply writing it as `/assets/ap-48.png` didn't work